### ISSUE TITLE
unknown field "tracing" in v1alpha1.ProxyConfig

### ISSUE THREAD

**Issue by esnible** [aspenmesh/istio-vet#42]
I ran `vet` on a 1.1 RC 2 cluster.

I did not dig into the cause of the problem.  Here is the relevant part of the output:

```
System pod count
================
INFO: 27 system pods out of mesh

E0305 16:13:12.410826   41454 util.go:209] Failed to parse yaml mesh config: unknown field "tracing" in v1alpha1.ProxyConfig
Vetter "MeshVersion" ran successfully and generated no notes
```

It isn't clear from the message which Vetter is complaining.  It isn't clear if the Vetter ran successfully and encountered this problem or if it failed and did not check the mesh config at all.

**Comment by nrjpoddar** [comment]
Did the vet binary/container exit after this error or went on to run other vetters?

**Comment by esnible** [comment]
I'll give you the full output from my small test cluster:

```
User pod count
==============
INFO: 11 user pods in mesh out of 15

System pod count
================
INFO: 27 system pods out of mesh

E0305 23:19:52.921450       1 util.go:209] Failed to parse yaml mesh config: unknown field "tracing" in v1alpha1.ProxyConfig
Vetter "MeshVersion" ran successfully and generated no notes

Vetter "MtlsProbes" ran successfully and generated no notes

Vetter "AppLabel" ran successfully and generated no notes

Missing prefix in service - myserver
====================================
WARNING: The service myserver in namespace default contains port names not prefixed with mesh supported protocols. Consider updating the service port name with one of the mesh recognized prefixes.

Missing prefix in service - t
=============================
WARNING: The service t in namespace default contains port names not prefixed with mesh supported protocols. Consider updating the service port name with one of the mesh recognized prefixes.

Vetter "serviceassociation" ran successfully and generated no notes

Vetter "DanglingRouteDestinationHost" ran successfully and generated no notes

Vetter "ConflictingVirtualServiceHost" ran successfully and generated no notes
```

**Comment by sdake** [comment]
I ran this tool on Istio 1.1.0.  Nice work folks!

I did notice this problem:
```
User pod count
==============
INFO: 1 user pods in mesh out of 1

System pod count
================
INFO: 25 system pods out of mesh

Vetter "MeshVersion" ran successfully and generated no notes

E0321 13:50:31.781274       1 util.go:209] Failed to parse yaml mesh config: unknown field "tracing" in v1alpha1.ProxyConfig
Vetter "MtlsProbes" ran successfully and generated no notes

Vetter "AppLabel" ran successfully and generated no notes

Vetter "serviceportprefix" ran successfully and generated no notes

Vetter "serviceassociation" ran successfully and generated no notes

Vetter "DanglingRouteDestinationHost" ran successfully and generated no notes

Vetter "ConflictingVirtualServiceHost" ran successfully and generated no notes```

Seems like a false positive.

Cheers
-steve

**Comment by francoispradeau** [comment]
@sdake @esnible This should be fixed with the latest, following #55. Do let me know if you see further issues!
